### PR TITLE
Update aws-china.md

### PR DESCRIPTION
**Pull request by pahud**
add [pahud/kops-cn](https://github.com/pahud/kops-cn) for reference. This will dramatically simplify the process of kops cluster creation in China.

### CODE REVIEW

**Comment by pahud** [comment]
I signed it

File: docs/aws-china.md [hunk 1]
```diff
@@ -297,3 +297,6 @@ It hasn't been tested as this approach was only a PR when the author experimenti
 [7]: https://docs.docker.com/registry/recipes/mirror/#use-case-the-china-registry-mirror
 [8]: https://github.com/kubernetes/kops/issues/3236
 [9]: https://aws.amazon.com/about-aws/global-infrastructure/
+
+### Other External Projects
```
Comment (idealhack) [inline docs/aws-china.md]: nit: I would like to see a blank line here

**Comment by idealhack** [comment]
@pahud,

To be clear, I appreciate the value of `kops-cn`.

But we need better documents about what's the differences between `kops-cn` and `kops`. In the meantime, forked projects are harder to maintain, do you have clear thoughts about this? Maybe integrate the features in `kops` is the way to go.

**Comment by pahud** [comment]
Hi @qqshfox 

As you are the owner of `aws-china.md` and this document is way out-of-date. I propose we update this document to break it down into two sections:

1. create Kops cluster with local public mirror
2. create Kops cluster from scratch with private mirror

As for the `public mirror` solution, I can add more description about [nwcdlabs/aws-cn](https://github.com/nwcdlabs/kops-cn) to help China users easily deploy Kops w/o the heavy-lifting while all of the previous content will be moved to the 2nd section for private mirror.

Thoughts?

**Comment by idealhack** [comment]
@pahud Thanks for your explaining here and privately with me. Now I have a better understanding of `kops-cn`. I hope some introduction can be written in this document.

Also, I would like to see some refactor for this document to serve users of AWS China better.

/cc @qqshfox @mingyuanzhu @leeeboo

**Comment by justinsb** [comment]
I actually think we be ready to link to other projects in the china-region case; it would be great to have those PRs contributed but it's really hard to support well in kops because I don't think anyone on the core team has access to the cn- regions.  We'll always try our best to review any PRs and they are very welcome, but it's not trivial to do always.

Because this is docs this can go in at any time in the release, it won't be compiled in to any project artifacts/releases.  Because we're in the burn-down for 1.11, I'm going to move this to 1.12 therefore.  I'd really like to understand what we're missing - some of the features that are good for China are also useful generally e.g. for people that want to run from their own private S3 mirror.

/milestone 1.12

**Pull request closed without merging**